docs/gettingstarted/structuring.md

```diff
@@ -25,6 +25,69 @@ What is `@Mod`?
 
 This is an annotation indicating to the Forge Mod Loader that the class is a Mod entry point. It contains various metadata about the mod. It also designates the class that will receive `@EventHandler` events. More information can be found at... (Coming Soon)
 
+All components of @Mod() are written inside the parantheses and chained together. There's only ever a single @Mod in your entire mod, sitting on top of your entry class.
```

Comment:
> Before the forge tutorial starts.

You say that like there aren't lots of people who teach themselves java by modding for Minecraft. They need every leg up they can get. :)
(It is quite dangerous to assume that Common Knowledge is a thing.)

> Multiple @Mods just mean that you have multiple mods.

Are there any practical applications here to have more than one @Mod annotation?